ocaml/xapi/xapi_clustering.ml

```diff
@@ -13,6 +13,7 @@
  *)
 
 open Cluster_interface
+open Db_cache_types
```

Comment:
Looks like there is one more `open` that should be moved.

Reply:
Looks like this one was left over + unused.